lib/fizzy/execute.cpp

```diff
@@ -356,6 +362,62 @@ T fnearest(T value) noexcept
         return t;
 }
 
+
+template <typename T>
+T fabs(T) noexcept = delete;
```

Comment:
I allowed myself to use template specialization what is more strict than previous attempt - no other type will be accepted. Please do the same for other functions.